modules/infrastructure-moving-monitoring.adoc

```diff
@@ -7,7 +7,7 @@
 = Moving the monitoring solution
 
 
-By default, the Prometheus Cluster Monitoring stack, which contains Prometheus, Grafana, and AlertManager, is deployed to provide cluster monitoring. It is managed by the Cluster Monitoring Operator. To move its components to different machines, you create and apply a custom config map.
+By default, the Prometheus Cluster Monitoring stack, which includes Prometheus, Thanos Querier, and AlertManager, is deployed to provide cluster monitoring. The monitoring stack is managed by the Cluster Monitoring Operator. To move its components to different machines, you create and apply a custom config map.
```

Comment:
Hi Brian. The current text is fine. But I started looking for ways to make a couple of improvements:
- Resolving the difference in terminology between the title and the abstract: "solution" vs. "stack"
- Using active instead of passive voice: "is deployed" > "deploys"
- Bringing the lede, "moving it components," to the fore.
- Providing a better term than "machines."

How about something like:
```suggestion
To <summary of benefits>, you can redeploy the Prometheus Cluster Monitoring stack to infrastructure nodes by using a custom config map.
```
OR, if the benefits are obvious or indeterminate:
```suggestion
To redeploy the Prometheus Cluster Monitoring stack to infrastructure nodes, you can create and apply a custom config map.
```

Reply:
Then, expand on the abstract in a separate paragraph:
```
By default, the Prometheus Cluster Monitoring stack includes Prometheus, Thanos Querier, and AlertManager. is deployed to provide cluster monitoring. The Cluster Monitoring Operator manages this stack.
```

Reply:
Using your suggestions, I rewrote the paragraph to read:

```
The monitoring stack includes multiple components, including Prometheus, Thanos Querier, and Alertmanager.
The Cluster Monitoring Operator manages this stack.
To redeploy the monitoring stack to infrastructure nodes, you can create and apply a custom config map.
```